src/typings.ts

```diff
@@ -15,19 +25,71 @@ export const supportedAuthnMethod = ['passwordless', 'sso', 'credentials'] as co
 export type AuthnMethod = typeof supportedAuthnMethod[number];
 
 export interface ConfigsOptions extends PrivateConfigsOptions {
+  /**
+   * Email used to pre-fill the email field in login or sign up or form.
+   *
+   * Set the default value via {@link MtLinkSdk.init}
+   */
   email?: string;
+  /**
+   * A redirection URL for redirecting a guest back to in the following condition:
+   * - Guest clicks on `Back to [App Name]` button in any Moneytree screen.
+   * - Guest refuses to give consent to access permission in the consent screen.
+   * - Guest logs out from Moneytree via an app with this client id
+   * - Revoke an app's consent from settings screen opened via an app with this client id
+   *
+   * ⚠️ No `Back to [App Name]` button will be shown if this value is not set, and any of the actions mentioned above will redirect the guest back to login screen by default.
+   *
+   * Set the default value via {@link MtLinkSdk.init}
+   */
   backTo?: string;
+  /**
+   * Show login or sign up screen when a session does not exist during an {@link MtLinkSdk.authorize} call.
+   *
+   * Set default value via {@link MtLinkSdk.init}
+   *
+   * @defaultValue 'login'
+   */
   authAction?: AuthAction;
+  /**
+   * If you wish to disable the login to sign up form toggle button and vice-versa in the auth screen, set this to `false`.
```

Comment:
I would suggest putting the button description in quotes, like on line 63. Otherwise this line is very hard to parse.